lib/src/util.dart

```diff
@@ -0,0 +1,6 @@
+class Tuple<T1, T2> {
```

Comment:
we could move this to realm_common_base.dart and delete the utils.dart

Reply:
Done